cogeo_mosaic/backends/base.py

```diff
@@ -64,3 +69,82 @@ def write(self):
     @abc.abstractmethod
     def update(self):
```

Comment:
Update signature to match children.

```python
    def update(
        self,
        features: Sequence[Dict],
        accessor: Callable,
        overwrite: bool = False,
        **kwargs: Any
    ):
```

Reply:
Maybe also have `accessor` default to `DEFAULT_ACESSOR`

Reply:
> Maybe also have accessor default to DEFAULT_ACESSOR

I was hesitant but yes